app/components/filters/include_new.rb

```diff
@@ -0,0 +1,13 @@
+module Filters
+  class IncludeNew
```

Comment:
This class is specific to Booking Facts but has been named as if it can be used for generically.